pkg/storage/store.go

```diff
@@ -252,7 +251,7 @@ func (s *store) storeForPeriod(p config.PeriodConfig, chunkClient client.Client,
 	)
 
 	// (Sandeep): Disable IndexGatewayClientStore for stores other than tsdb until we are ready to enable it again
-	/*if shouldUseBoltDBIndexGatewayClient(s.cfg) {
+	/*if shouldUseIndexGatewayClient(s.cfg) {
```

Comment:
I understand it's a comment, but still shouldn't this be now something like this?
```suggestion
	/*if s.cfg.BoltDBShipperConfig != nil && shouldUseIndexGatewayClient(s.cfg.BoltDBShipperConfig) {
```